codegen.mts

```diff
@@ -84,6 +84,31 @@ const config: CodegenConfig = {
           ID: 'string',
         },
         mappers: {
+          AuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          UserInvitedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          UserJoinedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          UserRemovedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          OrganizationSettingsUpdatedAuditLog:
+            '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          OrganizationTransferredAuditLog:
+            '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          ProjectCreatedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          ProjectSettingsUpdatedAuditLog:
+            '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          ProjectDeletedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          TargetCreatedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          TargetSettingsUpdatedAuditLog:
+            '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          TargetDeletedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          SchemaPolicySettingsUpdatedAuditLog:
+            '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          SchemaCheckedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          SchemaPublishAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          SchemaDeletedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          RoleCreatedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          RoleAssignedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          RoleDeletedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
+          RoleUpdatedAuditLog: '../modules/audit-logs/module.graphql.mappers#AuditLogMapper',
```

Comment:
Since `audit-log` is a new, self-contained module (i.e. doesn't `extend` any other types ), we could potentially avoid adding mappers to the `graphql-modules` 🙂. 

I've left a comment [here](https://github.com/kamilkisiela/graphql-hive/pull/5530/files#r1789975044) on how to do it for server preset (using `types.next.ts`)

Sorry that it's not really clear in the config at the moment - but we are very close to removing this old config altogether, so I didn't put any comments 🙏